ui/pages/confirm-transaction/conf-tx.js

```diff
@@ -135,7 +135,7 @@ class ConfirmTxScreen extends Component {
   }
 
   stopPropagation(event) {
-    if (event.stopPropagation) {
+    if (event?.stopPropagation) {
```

Comment:
Could we remove the `if` and just do `event?.stopPropagation?.()` or is that too gross?